apps/deploy-web/tests/ui/fixture/Leap/async/5330.js

```
@@ -250,42 +262,42 @@ _global.SENTRY_RELEASE = { id: "0.22.9" };
         d = Object.getOwnPropertySymbols,
         m = Object.prototype.hasOwnProperty,
         u = Object.prototype.propertyIsEnumerable,
-        p = (e, a, t) => (a in e ? i(e, a, { enumerable: !0, configurable: !0, writable: !0, value: t }) : (e[a] = t)),
-        f = (e, a) => {
-          for (var t in a || (a = {})) m.call(a, t) && p(e, t, a[t]);
-          if (d) for (var t of d(a)) u.call(a, t) && p(e, t, a[t]);
+        f = (e, a, t) => (a in e ? i(e, a, { enumerable: !0, configurable: !0, writable: !0, value: t }) : (e[a] = t)),
+        p = (e, a) => {
+          for (var t in a || (a = {})) m.call(a, t) && f(e, t, a[t]);
+          if (d) for (var t of d(a)) u.call(a, t) && f(e, t, a[t]);
```

Biome (2.1.2)

[error] Shouldn't redeclare 't'. Consider to delete it or rename it.

't' is defined here:


(lint/suspicious/noRedeclare)